src/Domain/Fixture/SubextensionManager.php

```diff
@@ -206,6 +206,31 @@ public function findDevDependenciesByPackage(Package $package): array {
     }
   }
 
+  /**
+   * Finds the plugins configured in allow-plugins config of a given package.
+   *
+   * @param \Acquia\Orca\Domain\Package\Package $package
+   *   The package to search for plugins.
+   *
+   * @return array
+   *   An indexed array of all the dev-dependencies.
+   *
+   * @throws \Acquia\Orca\Exception\OrcaFileNotFoundException
+   * @throws \Acquia\Orca\Exception\OrcaException
+   * @throws \Acquia\Orca\Exception\OrcaParseError
+   */
+  public function findAllowPluginsByPackage(Package $package): array {
+    $parent_path = $package->getInstallPathAbsolute();
+    try {
+      $config = $this->configLoader->load("$parent_path/composer.json");
+      $allow_plugins = $config->get("config.allow-plugins");
+      return $allow_plugins ? array_keys($allow_plugins) : [];
```

Comment:
`$config->get()` takes a default value, so you can save a step here:

```suggestion
      $allow_plugins = $config->get("config.allow-plugins", []);
      return array_keys($allow_plugins);
```